modules/modules.nix

```diff
@@ -148,6 +148,7 @@ let
     ./programs/zsh.nix
     ./programs/zsh/prezto.nix
     ./services/barrier.nix
+    ./services/betterlockscreen.nix
```

Comment:
Please add a news item for this new module.

Reply:
Done, thanks :)